include/ginkgo/core/matrix/permutation.hpp

```diff
@@ -114,6 +114,23 @@ class Permutation : public EnableLinOp<Permutation<IndexType>>,
         return permutation_.get_num_elems();
     }
 
+    /**
+     * Get the permute masks
+     *
+     * @param permute_mask the permute masks
+     */
+    mask_type get_permute_mask() { return enabled_permute_; }
```

Comment:
Maybe const makes sense here.
```suggestion
    mask_type get_permute_mask() const { return enabled_permute_; }
```